Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -657,11 +657,27 @@ private class PhotonActionSheetCell: UITableViewCell {
         accessibilityLabel = action.title
         selectionStyle = action.handler != nil ? .default : .none
 
-        if let iconName = action.iconString, let image = UIImage(named: iconName)?.withRenderingMode(.alwaysTemplate) {
-            statusIcon.sd_setImage(with: action.iconURL, placeholderImage: image, options: []) { (img, err, _, _) in
-                if let img = img {
-                    self.statusIcon.image = img.createScaled(PhotonActionSheetUX.IconSize)
-                    self.statusIcon.layer.cornerRadius = PhotonActionSheetUX.IconSize.width / 2
+        let button = TabsButton().clone()
+        let renderer = UIGraphicsImageRenderer(size: button.bounds.size)
+        let tabImage = renderer.image { ctx in
+            button.drawHierarchy(in: button.bounds, afterScreenUpdates: true)}.withRenderingMode(.alwaysTemplate)
+
+        if let iconName = action.iconString {
+            if iconName == "tabs-button" {
+                let image = tabImage //uses UIImage from TabsButton
+                statusIcon.sd_setImage(with: action.iconURL, placeholderImage: image, options: []) { (img, err, _, _) in
```

Comment:
The original code used `sd_setImage` so I see it got re-used. That is for downloading images, we can look into why it is being used here.

Reply:
(The code does say `iconURL`, so I guess that isn't a file URI, but I am at a loss to recall what we would be downloading at this point in the code). I think having an image in-hand (the `tabImage` and then calling sd_setImage to download an image for `iconURL` is confusing, likely isn't working as expected.